deployment/terraform/app.tf

```diff
@@ -1,121 +1,50 @@
+    api_server_image_url = "896538046175.dkr.ecr.us-east-1.amazonaws.com/rastervision-api-server:${var.image_version}"
     region               = "${var.aws_region}"
     environment          = "${var.environment}"
   }
 }
 
-resource "aws_ecs_task_definition" "app_server" {
-  family                = "${var.project_id}_${var.environment}AppServer"
-  container_definitions = "${data.template_file.app_server_http_ecs_task.rendered}"
+resource "aws_ecs_task_definition" "raster_vision" {
+  family                = "${var.environment}RasterVision"
+  container_definitions = "${data.template_file.raster_vision_ecs_task.rendered}"
 }
 
-# Defines running an ECS task as a service
-resource "aws_ecs_service" "server" {
-  name                               = "${var.project_id}-${var.environment}Server"
-  cluster                            = "${aws_ecs_cluster.container_instance.id}"
-  task_definition                    = "${aws_ecs_task_definition.app_server.family}:${aws_ecs_task_definition.app_server.revision}"
-  desired_count                      = "${var.desired_instance_count}"
-  iam_role                           = "${aws_iam_role.container_instance_ecs.name}"
-  deployment_minimum_healthy_percent = "0"                                                                                           # allow old services to be torn down
-  deployment_maximum_percent         = "100"
+resource "aws_cloudwatch_log_group" "raster_vision" {
+  name = "log${var.environment}RasterVision"
 
-  load_balancer {
-    target_group_arn = "${aws_alb_target_group.server_app_https.id}"
-    container_name   = "nginx"
-    container_port   = "80"
+  tags {
+    Environment = "${var.environment}"
   }
 }
+
+module "raster_vision_ecs_service" {
+  source = "github.com/azavea/terraform-aws-ecs-web-service?ref=0.2.0"
+
+  name                = "RasterVision"
+  vpc_id              = "${data.terraform_remote_state.core.vpc_id}"
+  public_subnet_ids   = ["${data.terraform_remote_state.core.public_subnet_ids}"]
+  access_log_bucket   = "${data.terraform_remote_state.core.logs_bucket_id}"
+  access_log_prefix   = "ALB/RasterVision"
+  port                = "443"
+  ssl_certificate_arn = "${var.ssl_certificate_arn}"
+
+  cluster_name                   = "${data.terraform_remote_state.core.container_instance_name}"
+  task_definition_id             = "${aws_ecs_task_definition.raster_vision.family}:${aws_ecs_task_definition.raster_vision.revision}"
+  desired_count                  = "${var.raster_vision_ecs_desired_count}"
+  min_count                      = "${var.raster_vision_ecs_min_count}"
+  max_count                      = "${var.raster_vision_ecs_max_count}"
+  deployment_min_healthy_percent = "${var.raster_vision_ecs_deployment_min_percent}"
+  deployment_max_percent         = "${var.raster_vision_ecs_deployment_max_percent}"
+  container_name                 = "nginx"
+  container_port                 = "443"
```

Comment:
An explicit `health_check_path` should probably be set here to avoid an issue where defaults don't work in the future.